addons/web_dashboard/controllers/main.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+
+from openerp import http, _
+from openerp.http import request
+from openerp.exceptions import RedirectWarning
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class Dashboard(http.Controller):
+
+    def get_users_list(self):
```

Comment:
call res.users (if you need you add a mtehod on res.users ) web_dashboard_get_XXXXX